.github/workflows/sycl-windows-build.yml

```diff
@@ -76,6 +76,13 @@ jobs:
     outputs:
       build_conclusion: ${{ steps.build.conclusion }}
     steps:
+    - name:  Detect hung tests
+      if: always()
+      shell: powershell
+      run: |
+        Invoke-WebRequest -Uri "https://raw.githubusercontent.com/intel/llvm/refs/heads/sycl/devops/scripts/windows_detect_hung_tests.ps1" -OutFile "windows_detect_hung_tests.ps1"
+        powershell.exe -File windows_detect_hung_tests.ps1
+        exit $LASTEXITCODE
```

Comment:
I think this should *NOT* result in any failure.

Reply:
couple of points here

1) the single place we call it today already does
2) im worried we will never catch hung tests if we dont fail because otherwise the only way to catch it would be to manually read the ci logs